src/main/java/com/aws/greengrass/deployment/ShadowDeploymentListener.java

```diff
@@ -87,6 +87,11 @@
     public static final String DEVICE_OFFLINE_MESSAGE = "Device not configured to talk to AWS Iot cloud. "
             + "Single device deployment is offline";
     public static final String SUBSCRIBING_TO_SHADOW_TOPICS_MESSAGE = "Subscribing to Iot Shadow topics";
+
+    private static final String SHADOW_UPDATE_ACCEPTED_TOPIC = "$aws/things/{thingName}/shadow/update/accepted";
```

Comment:
We switched to named shadows, so the topic prefix would be aws/things/{thingName}/shadow/name/AWSManagedGreengrassV2Deployment

Reply:
Updated